bimmer_connected/api/authentication.py

```diff
@@ -301,6 +325,7 @@ async def _refresh_token_china(self):
                 # Try logging in using refresh_token
                 response = await client.post(
                     AUTH_CHINA_TOKEN_URL,
+                    headers={"x-login-nonce": generate_cn_nonce(self.gcid)},
```

Comment:
Where do we get `gcid` on first try with refresh token?

Reply:
Yes, this is a problem. Before logging in, we don't know what the current GCID is. Currently, it seems that the GCID for each account remains the same, but this is only for now. Therefore, it's possible that the refresh token cannot be used for direct login in the Chinese region temporarily.

Reply:
Currently, adding `gcid` to the init parameter of MyBMWAuthentication.